docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+# initial docker-compose file. comments added from docker docs
+
+version: "3"
+services:
+  web:
+    # replace username/repo:tag with your name and image details
+    image: tylergreene/secret-ui-docker:latest
+    deploy:
+      # run 5 instances of this image as a service called "web"
+      replicas: 5
+      resources:
```

Comment:
interesting, I've rarely seen anyone limit resources in docker-compose. 
Why'd you choose to do this? Just to test it out or did you read some best practices somewhere? 
Have you run into any performance issues? I suppose not yet since the app is just starting out.

Reply:
reading through some of the setup docs, i saw it suggested as an option, mostly just wanted to try it out and see if it worked. i'm not sure if there will be any real reason to keep these (or any) limits on the project in the future. would it be correct to say that if one instance of the container was overloaded, another instance would receive the traffic?